config/clusters/hhmi/spyglass.values.yaml

```diff
@@ -41,14 +50,20 @@ jupyterhub:
   singleuser:
     initContainers: []
     storage:
-      # No persistent storage should be kept to reduce any potential data
-      # retention & privacy issues.
       type: none
-      extraVolumeMounts: []
+      extraVolumes:
+        - name: shared-dir-pv
+          persistentVolumeClaim:
+            claimName: home-nfs
+      extraVolumeMounts:
+        - name: shared-dir-pv
+          mountPath: /home/jovyan/shared-readonly
+          subPath: _shared
+          readOnly: true
     defaultUrl: /lab
     image:
-      name: quay.io/lorenlab/hhmi-spyglass-image
-      tag: "c307f9418a60"
+      name: quay.io/2i2c/hhmi-spyglass-image
+      tag: "8be848d09076"
```

Comment:
Unrelated change - I think it may be reverting something new for something old - right?

Reply:
@consideRatio yes, unrelated but necessary - I put this in the description:

> I was asked to bump an image tag for this by @jmunroe and have done so.

I couldn't keep it separate because it was needed for him to test his changes.